app/services/appropriate_bodies/claim_an_ect/register_ect.rb

```diff
@@ -38,7 +38,8 @@ def create_or_update_teacher
         teacher.assign_attributes(
           trs_first_name: pending_induction_submission.trs_first_name,
           trs_last_name: pending_induction_submission.trs_last_name,
-          trs_qts_awarded_on: pending_induction_submission.trs_qts_awarded_on
+          trs_qts_awarded_on: pending_induction_submission.trs_qts_awarded_on,
+          trs_initial_teacher_training_provider_name: pending_induction_submission.trs_initial_teacher_training_provider_name
```

Comment:
Is this something we will eventually record a change event for?

Reply:
@edujackedu the reason I ask is I am doing a refactor within `RegisterECT` so you might have feedback when it's ready. @peteryates is the goal to track events for everything eventually?